### ISSUE TITLE
Compatibility with Woocommerce Subscriptions?

### ISSUE THREAD

**Issue by rai-acuariopro** [cyberwombat/woocommerce-fastspring-payment-gateway#33]
Hi there,
This month I was able to test the plugin. All working well with normal transactions, thanks!
But is a not working well with Subscriptions from FastSpring or from Woocommerce Subscriptions, the order is mark as pending payment or cancel but was already made, the status does not update itself. And then the Woocommerce Subscriptions is lock tu activate.

I assume this plugin was compatible with Woocommerce Subscriptions, is it? not sure why I assume that.

And I was wondering how much money it is need it to update this plugin and make it compatible with this plugin.

Thanks again!

**Comment by cyberwombat** [comment]
Yes it supports https://woocommerce.com/products/woocommerce-subscriptions/  plugin though not tested with latest.  Have you turned debug on? a log will help in this https://github.com/cyberwombat/woocommerce-fastspring-payment-gateway/wiki/Troubleshooting - I am not actively working on plugin anymore but I can try to help decipher your logs.

**Comment by rai-acuariopro** [comment]
Thanks for your reply,
I have the logs on but there is nothing there, should be there listed with
the name of the plugin right?
Anyway I don´t find anything there.



Atentamente,
Rai Singh Uriarte
*Acuario Pro* Online Marketing

**Comment by cyberwombat** [comment]
If you follow the wiki instructions to turn debug on etc it will show all the webhook calls from FS back to you and potentially any errors it encounters - perhaps its not finding the subscription order etc.. But if you set it up correctly there **will** be some kind of entries in the log

**Comment by rai-acuariopro** [comment]
Thanks, I'll check it out.

**Comment by rai-acuariopro** [comment]
Yashua,
I check the webhooks and everything looks fine but I can not find any error
log-in woo. Also, the subscriptions keep failing in woo to create the
proper workflow. The people is successful in paying but when is a
subscription the payment in woo many times is mark as cancelled, not the
same when is a normal product purchased, then it never happens this. So I
can assure you that the stability with subscriptions is not working. I even
have technical people from woo to check on the issue and they all point
that the connection between the plugin and fastspring is not working.
Just want to let you know. Thanks.

Atentamente,
Rai Singh Uriarte
*Acuario Pro* Online Marketing
